src/Compilers/Test/Core/Compilation/TestOperationVisitor.cs

```diff
@@ -1284,6 +1284,7 @@ public override void VisitPropertySubpattern(IPropertySubpatternOperation operat
 
         public override void VisitSwitchExpression(ISwitchExpressionOperation operation)
         {
+            Assert.True(operation.IsExhaustive || true);
```

Comment:
>Assert.True(operation.IsExhaustive || true); [](http://example.com/codeflow?start=12&length=44)

Not a big issue, but it doesn't feel necessary to assert this. It should be sufficient to query the property. Something like ```_ = operation.IsExhaustive```. See an example in the method below.